website/src/css/pages/_jobs.scss

```diff
@@ -245,6 +245,42 @@
         }
     }
 
+    .reviews-carousel {
```

Comment:
I would try to see how many of these styles you can set using Bootstrap utility classes in the `tsx` itself. We've got custom heights and max-widths now as well! Using the built-in breakpoints will let you avoid writing media queries in many cases as well! So you could do `px-lg-5 px-md-3 px-0` to set horizontal padding on an element at the set breakpoints, for example.

Reply:
Thanks! Can you explain our custom heights and max-widths? I had a hard time finding the documentation on Bootstrap. I'm using the utility classes for all the background & spacing styles here now though 😄

Reply:
Yay! Awesome :)

Agreed that the custom stuff isn't well-documented! For height, you can do `height-<whatever>` (including `auto`) — there are defined breakpoints as well, which align with Bootstrap defaults. The `_spacing` file will show you what's available.

For max width, it would be `mw-<whatever>` — no defined breakpoints, but feel free to add them if you need them!

Reply:
Neat, thank you so much! Okay this change should be good now :)